packages/hint/docs/user-guide/telemetry/index.md

```diff
@@ -0,0 +1,59 @@
+# Telemetry
+
+## What are we tracking
+
+At the end of the second time you run `webhint` we will ask you if you want to
```

Comment:
Let's simplify this a bit and maybe use:

> At the end of the second run `webhint` we will ask if you want to
send `limited usage information` to help us to build a better product.